roles/kubespray-defaults/vars/main.yml

```diff
@@ -1,2 +1,7 @@
 ---
+# Kubespray constants
+
 kube_proxy_deployed: "{{ 'addon/kube-proxy' not in kubeadm_init_phases_skip }}"
+
+# The lowest version allowed to upgrade from (same as calico_version in the previous branch)
+calico_min_version_required: "v3.15.2"
```

Comment:
roles/download/defaults/main.yml contains v3.13.3 as https://github.com/kubernetes-sigs/kubespray/blob/c0fd5b2e8463e92a7732d97aabf4d5f62dc76330/roles/download/defaults/main.yml#L370
I guess we can remove the above v3.13.3 from the file.

Reply:
yes and v3.14.1 since only install of v3.15.2 is allowed with this new strategy

Reply:
Thanks for updating.